frontend/components/panels/VaultLiquidationPanel.vue

```diff
@@ -15,15 +15,15 @@
                 @execute="executeWithOtherWallet"
             />
             <BaseButton
-                :disabled="!isLiquidatable || disabled"
+                :disabled="disabled"
                 :is-loading="isLiquidating"
                 @click="isExecuteToAnotherWalletModalShown = true"
             >
                 Liquidate to another wallet
             </BaseButton>
             <BaseButton
                 type="primary"
-                :disabled="!isLiquidatable || disabled"
+                :disabled="disabled"
                 :is-loading="isLiquidating"
                 @click="$emit('liquidate', walletAddress)"
             >
```

Comment:
Why was it removed? If the vault is not in `liquidatable` state it can not be removed, right?

Reply:
This change was made by @aomafarag. I do not know either, maybe there was a reason we don't see?

Reply:
No, [specified line only checks for the limits](https://github.com/sidestream-tech/unified-auctions-ui/blob/1e885274400b0348d2f473d6f19910b96a2675d3/frontend/components/vault/VaultLiquidationTransactionFlow.vue#L50), but it's (obviously) not sufficient to liquidate a specific vault if the limits are not reached. If it would be true any current vault could've been liquidated right now